packages/pigeon/platform_tests/test_plugin/windows/test_plugin.cpp

```diff
@@ -90,18 +90,18 @@ ErrorOr<AllTypes> TestPlugin::EchoAllTypes(const AllTypes& everything) {
 ErrorOr<std::optional<AllNullableTypes>> TestPlugin::EchoAllNullableTypes(
     const AllNullableTypes* everything) {
   if (!everything) {
-    return std::nullopt;
+    return std::optional<AllNullableTypes>(std::nullopt);
```

Comment:
The changes in this file made me sad. I'm assuming the standard doesn't allow the two-level chaining of implicit conversions that this was relying on (foo -> optional<Foo> -> ErrorOr<optional<Foo>>), so this does one of them explicitly and then relies on the implicit construction of `ErrorOr`.